docs/src/main/asciidoc/funqy-http.adoc

```diff
@@ -0,0 +1,106 @@
+////
+This guide is maintained in the main Quarkus repository
+and pull requests should be submitted there:
+https://github.com/quarkusio/quarkus/tree/master/docs/src/main/asciidoc
+////
+= Quarkus - Funqy HTTP
+
+include::./attributes.adoc[]
+:extension-status: preview
+
+The guide walks through quickstart code to show you how you can deploy and invoke on Funqy functions in a pure HTTP environment.
+
+== Prerequisites
+
+To complete this guide, you need:
+
+* less than 15 minutes
+* Read about link:funqy[Funqy Basics].  This is a short read!
+* an IDE
+* JDK 1.8+ installed with `JAVA_HOME` configured appropriately
+* Apache Maven {maven-version}
+
+== The Quickstart
+
+Clone the Git repository: `git clone {quickstarts-clone-url}`, or download an {quickstarts-archive-url}[archive].
+
+The solution is located in the `funqy-http-quickstart` {quickstarts-tree-url}/funqy-quickstarts/funqy-http-quickstart[directory].
+
+== The Code
+
+If you look at the Java code, you'll see that there is no HTTP specific API.  Its just simple Java methods
+annotated with `@Funq`.  Simple, easy, straightforward.
+
+== Build Project
+
+[source, shell]
+----
+mvn clean quarkus:dev
+----
+
+This starts your functions in Quarkus dev mode.
+
+== Execute Funqy HTTP functions
+
+Funqy HTTP input and output is always JSON.  The Jackson JSON parser is used behind the scenes and you can
+use Jackson annotations to fine tune your JSON mappings for your input and output objects.  We do not recommend
+using Jackson annotations though.  If we find a JSON parser that's better than Jackson we will use it!  So try
```

Comment:
`If we find a JSON parser that's better than Jackson we will use it` sound a little vague, what does it mean in practice?